prova-react/src/features/UI/UI.jsx

```diff
@@ -8,14 +8,22 @@ import SidebarContext from "../../common/SidebarContext";
 
 const UI = ({ controlsRef }) => {
     const { isSidebarVisible, setIsSidebarVisible } = useContext(SidebarContext);
+    const [ready,setReady] = useState(false);
```

Comment:
non mi piace il ritardo prima di caricare UI carrello.. provare un' altra soluzione se possibile
esempio provare a mettere [canvasLoad,setCanvasLoad]=useState(false) nel componente App.. successivamente mettere in <Canvas  onRender={setCanvasLoad(true)}/> per poi {canvasLoad && <UI />}